### PR TITLE
Update ElasticSearch client to fix vulnerability

### PR DESCRIPTION
**Pull request by solth**
Update ElasticSearch client library to prevent potential arbitrary code execution exploit.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -65,7 +65,7 @@
         <saxon.version>9.9.1-8</saxon.version>
         <log4j.version>2.19.0</log4j.version>
         <junit.version>5.9.2</junit.version>
-        <elasticsearch.version>7.17.7</elasticsearch.version>
+        <elasticsearch.version>7.17.8</elasticsearch.version>
```
Comment (henning-gerhardt) [inline pom.xml:68]: Why we are not switching to latest release aka 7.17.10 (at 05/05/2023) of elastic search?

Comment (solth) [reply pom.xml:68]: I just used and tested the first newer version that was marked as fixing the mentioned vulnerability to minimize the potential for incompatibilities, but I guess if ElasticSearch adheres to the semantic versioning guidelines, 7.17.10 should work, too.

Comment (solth) [reply pom.xml]: Just tested it and version 7.17.10 does indeed work as well without problems, so I changed the version in the pull request as you suggested.

**Review by henning-gerhardt — APPROVED** [review]
I tested the new client version with an old (7.17.7) and new (7.17.10) ES server and I got no issues: searching, updating, indexing. But I tested this only with a few data and not with our complete data set.

**Pull request merged**